openslides/utils/views.py

```diff
@@ -45,3 +48,22 @@ def method_call(self, request: Any, *args: Any, **kwargs: Any) -> Any:
     # Add the http-methods and delete the method "method_call"
     get = post = put = patch = delete = head = options = trace = method_call
     del method_call
+
+
+class IndexView(View):
```

Comment:
The wording `IndexView` is uses twice: Here for a general view class and in core/views for the real index view. Maybe change this to `TemplateView`.

Reply:
Done.